platform/app/prisma/migrations/20260825120000_experiment_version_counter_version/migration.sql

```
@@ -0,0 +1,37 @@
+-- Every existing row was written at the counter it carries in "version", so
+-- the backfill is that equality.
+--
+-- To roll back, uncomment and run manually. Dropping the column loses the
+-- recency of the rolling autosave row; no other data is affected.
+-- DROP INDEX "ExperimentVersion_projectId_experimentId_counterVersion_idx";
+-- ALTER TABLE "ExperimentVersion" DROP COLUMN "counterVersion";
+
+-- AlterTable
+ALTER TABLE "ExperimentVersion" ADD COLUMN "counterVersion" INTEGER;
+
+UPDATE "ExperimentVersion" SET "counterVersion" = "version" WHERE "counterVersion" IS NULL;
```

SQLFluff (4.3.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)